pulsar-client-kafka-compat/pulsar-client-kafka/src/main/java/org/apache/kafka/clients/consumer/PulsarKafkaConsumer.java

```diff
@@ -159,6 +163,16 @@ private PulsarKafkaConsumer(ConsumerConfig config, Deserializer<K> keyDeserializ
         }
     }
 
+    private OffsetResetStrategy getStrategy(final String strategy) {
+    	if (strategy.equals("earliest")) {
+    		return OffsetResetStrategy.EARLIEST;
+    	} else if (strategy.equals("latest")) {
+    		return OffsetResetStrategy.LATEST;
+    	} else {
```

Comment:
I would prefer failing-fast if strategy is not null or empty here and logging a message to tell users that this is a wrong setting.

Reply:
Oh, actually in Kafka, OffsetResetStrategy.NONE in fact is a viable option for users. In this case, if its NONE when the offset has an invalid position, KafkaConsumer will throw an error. I forgot to replicate this behavior in the code. Will have to do that.

Reply:
Sure, will do.